circe-color-nicks.el

```diff
@@ -81,18 +81,22 @@ See `enable-circe-color-nicks'."
 recommended is 7:1, or at least 4.5:1 (7 stands for 7:1 here).
 Lower value allows higher color spread, but could lead to less
 readability."
+  :type 'number
```

Comment:
I've thought so too but then I checked that these options are compared with `circe-color-diff` which does return a float. So it seems like float values for these should work as well. Also, the description for some of these mentions float examples like `4.5:1`. If only integers are allowed, perhaps that example should be clarified.

Will fix the type of indeed only integers are allowed.